scheds/rust/scx_mitosis/src/bpf/mitosis.bpf.c

```diff
@@ -1487,9 +1478,9 @@ void BPF_STRUCT_OPS(mitosis_dump_task, struct scx_dump_ctx *dctx,
 		return;
 
 	scx_bpf_dump(
-		"Task[%d] vtime=%llu basis_vtime=%llu cell=%u dsq=%x all_cell_cpus_allowed=%d\n",
+		"Task[%d] vtime=%llu basis_vtime=%llu cell=%u dsq=%llu all_cell_cpus_allowed=%d\n",
```

Comment:
If you're printing the raw value I think I prefer hex - it's easier to read, no?